README.md

```diff
@@ -0,0 +1,87 @@
+# Hopsan
+
+Hopsan is a free open-source multi-domain system simulation tool developed at the division of Fluid and mechatronic systems at Linkoping university.
+
+Features include:
+
+* Simulation core library
+    * Plain C++ library for easy integration
+    * Multi-core support for faster simulations
+    * Create your own component models libraries in C++
```

Comment:
Yes line breaks are tricky. Apparently the line should end in two white spaces which is easy to forget